### PR TITLE
Prefix hollow-cluster-master name with instance prefix

### PR DESCRIPTION
**Pull request by gmarek**

### CODE REVIEW

**Comment by karlkfi** [comment]
looks like etcd failed to dl on that smoke test. rerunning.

**Comment by gmarek** [comment]
@quinton-hoole PTAL

File: test/kubemark/start-kubemark.sh [hunk 1]
```diff
@@ -43,7 +43,8 @@ make
 rm kubemark
 cd $CURR_DIR
 
-MASTER_NAME="hollow-cluster-master"
+MASTER_NAME="${INSTANCE_PREFIX:-"default"}-kubemark-master"
```
Comment (quinton-hoole) [inline test/kubemark/start-kubemark.sh]: Shouldn't need the "default here, as it's already specified in config-default.sh, which is sourced above.

File: test/kubemark/start-kubemark.sh [hunk 2]
```diff
@@ -62,14 +63,14 @@ gcloud compute instances create "${MASTER_NAME}" \
     --scopes "storage-ro,compute-rw,logging-write" \
     --disk "name=${MASTER_NAME}-pd,device-name=master-pd,mode=rw,boot=no,auto-delete=no"
 
-gcloud compute firewall-rules create "kubemark-master-https" \
+gcloud compute firewall-rules create "${INSTANCE_PREFIX:-"default"}-kubemark-master-https" \
```
Comment (quinton-hoole) [inline test/kubemark/start-kubemark.sh]: Same comment as above.

File: cluster/kubemark/config-default.sh [hunk 3]
```diff
@@ -33,9 +33,9 @@ MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-container-vm-v20150806}
 MASTER_IMAGE_PROJECT=${KUBE_GCE_MASTER_PROJECT:-google-containers}
 
 NETWORK=${KUBE_GCE_NETWORK:-default}
-INSTANCE_PREFIX="hollow-kubernetes"
-MASTER_NAME="hollow-cluster-master"
-MASTER_TAG="hollow-cluster-master"
+INSTANCE_PREFIX="${INSTANCE_PREFIX:-"default"}"
```
Comment (quinton-hoole) [inline cluster/kubemark/config-default.sh]: Why not default to "kubemark", or perhaps $CLUSTER_NAME or $E2E_CLUSTER_NAME ?

**Comment by quinton-hoole** [comment]
nits, then LGTM.

File: test/kubemark/start-kubemark.sh (hunk 1 above)
Comment (gmarek) [inline test/kubemark/start-kubemark.sh]: Done.

File: test/kubemark/start-kubemark.sh (hunk 2 above)
Comment (gmarek) [inline test/kubemark/start-kubemark.sh]: Done.

File: cluster/kubemark/config-default.sh (hunk 3 above)
Comment (gmarek) [inline cluster/kubemark/config-default.sh]: E2E_CLUSTER_NAME is set only by Jenkins - I'd rather have some sane default without using environmental variables, if possible.

Comment (gmarek) [inline cluster/kubemark/config-default.sh]: And kubemark-kubemark-master just sounds bad;)

**Comment by quinton-hoole** [comment]
LGTM

**Pull request merged**